src/apps/backend/modules/common/proxy.py

```diff
@@ -0,0 +1,18 @@
+from flask import Flask
+from werkzeug.middleware.proxy_fix import ProxyFix
+from modules.config.config_service import ConfigService
+
+class Proxy:
+    @staticmethod
+    def apply_proxy_if_enabled(app: Flask) -> Flask:
```

Comment:
This does not should be in `common`, only applicable for `server.py`. Consider inlining the code there itself for now.